proposals/lambda-discard-parameters.md

```diff
@@ -0,0 +1,21 @@
+# Lambda discard parameters
+
+## Summary
+
+Allow discards (`_`) to be used as lambda parameters.
+For example:
+- `(_, _) => 0`
+- `(int _, int _) => 0`
+- `delegate(int _, int _) { return 0; }`
+
+## Motivation
+
+Unused lambda parameters do not need to be named. The intent of discards is clear, i.e. they are unused/discarded.
+
+## Detailed design
+
+In a lambda or delegate declaration with more than one parameter named `_`, those parameters are discard parameters.
+
+Discard parameters are not in scope in the body of the lambda or delegate. So they cannot be accessed.
```

Comment:
>parameters are not in scope [](start = 8, length = 27)

Can you pose this as a set of changes to the language specification?

Reply:
I took a stab at this:

In the section on [method parameters](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#method-parameters), I would say that discard parameters do not add any names in scope.
In the section on [corresponding parameters](https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#corresponding-parameters), I would say that named arguments do not correspond to discard parameters.